tests/server/sws.c

```diff
@@ -1319,6 +1319,14 @@ static curl_socket_t connect_to(const char *ipaddr, unsigned short port)
   }
 #endif
 
+  if(0 != curlx_nonblock(serverfd, TRUE)) {
```

Comment:
Please add a comment why this is being set to nonblocking. It's not obvious and it would be easy for someone to see the select() loop below and just switch the connect back to blocking.

Reply:
will do.